docs-client/src/containers/MethodPage/index.tsx

```diff
@@ -157,6 +153,14 @@ const MethodPage: React.FunctionComponent<Props> = (props) => {
     debugTransport !== undefined &&
     debugTransport.supportsMimeType(GRAPHQL_HTTP_MIME_TYPE);
 
+  const parameterVariables = method.parameters.map((param) => {
+    const childFieldInfos = props.specification.getStructByName(
+      param.typeSignature,
+    )?.fields;
+    if (childFieldInfos) return { ...param, childFieldInfos };
+    return param;
```

Comment:
nit: For consistency. 
```suggestion
    if (childFieldInfos) {
       return { ...param, childFieldInfos };
    }
    return param;
```